### PR TITLE
[docs] Arctype integration documentation

### PR DESCRIPTION
**Pull request by aatrey-yb**
@netlify /latest/tools/arctype/

### CODE REVIEW

**Comment by aatrey-yb** [comment]
@netlify docs/content/latest/integrations/third party tools/arctype.md
Arctype docs

File: docs/content/latest/tools/arctype.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Before you begin
+
+Your YugabyteDB cluster should be up and running. If you're new to YugabyteDB, create a local cluster in less than five minutes following the steps in [Quick Start](../../../quick-start/install). You can also get started with free tier of [YugabyteDB Fully-Managed Cloud](https://www.yugabyte.com/cloud/). You also need to install Arctype client on your computer. Arctype clients are available for Windows, Linux and Mac that can be downloaded from [Arctype](https://arctype.com/) website.
+
+## Create a database connection
+
+After installing and launching Arctype client on your computer, follow the steps below to connect to YugabyteDB.
+
+1. Create an Arctype account and login after launching the client.
+2. On 'Connect a Database' step, select PostgreSQL and click Continue.
```
Comment (FranckPachot) [inline docs/content/latest/tools/arctype.md:33]: They have now a dedicated YugabyteDB connection type. Better to mention this one (an show it in screenshot)

File: docs/content/latest/tools/arctype.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+1. Enter your database name, username, password, and nickname for this database, and click Continue.
+
+    ![Enter database connection details](/images/develop/tools/arctype/arctype-connect-step5.png)
+
+1. Enter a workspace name, and click Continue.
+
+You've successfully created a connection to your YugabyteDB database, and you can now start querying and visualizing your DB using Arctype.
+
+## What's Next
```
Comment (polarweasel) [inline docs/content/latest/tools/arctype.md:49]: As the the last step in "Create a database connection", you need to include an actual test of your connection. See, for example, the last step in the [DBeaver doc](https://docs.yugabyte.com/latest/tools/dbeaver-ysql/). When I get to the end of the steps, I want to know that I've been successful!

**Review by polarweasel — CHANGES_REQUESTED** [review]
This is looking good! I've made some edits in your branch, and left a couple of comments.

One further comment: you probably should blur out the IP addresses and domain names, since those appear to be publicly accessible.

**Comment by aatrey-yb** [comment]
https://github.com/netlify /latest/tools/arctype/

File: docs/content/latest/tools/arctype.md (hunk 2 above)
Comment (aatrey-yb) [reply docs/content/latest/tools/arctype.md:47]: added test connection step

File: docs/content/latest/tools/arctype.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+    ![Connect DB Step](/images/develop/tools/arctype/arctype-conect_step3.png)
+
+1. Enter your YugabyteDB host and port information, and click Continue.
+
+    ![Enter host and port](/images/develop/tools/arctype/arctype-connect-step4.png)
+
+1. Enter your database name, username, password, and nickname for this database, and click Continue.
+
+    ![Enter database connection details](/images/develop/tools/arctype/arctype-connect-step5.png)
```
[the remark this answers is not in the document]
Comment (aatrey-yb) [reply docs/content/latest/tools/arctype.md]: new screenshots have local IP(127). Added a note to follow cert instructions if using Yugabyte Cloud.

File: docs/content/latest/tools/arctype.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    Note: If you're using YugabyteDB Cloud, you need to add your computer to the cluster IP allow list. Refer to [Assign IP Allow Lists](https://docs.yugabyte.com/latest/yugabyte-cloud/cloud-secure-clusters/add-connections/).
+    You also need to download and install CA Cert root.crt certificate on your computer from YugabyteDB Cloud console for TLS encryption.
+    
+    {{< /note >}}
+
+4. Enter your YugabyteDB host, port, database, user and password information, and click 'Test Connection' and save if connection is successful.
+
+    ![Enter host and port](/images/develop/tools/arctype/arctype-connect-step4.png)
+
+5. You can see the schemas and tables availabe in the YugabyteDB in the navigation panel.
```
Comment (polarweasel) [inline docs/content/latest/tools/arctype.md:48]: Fix up formatting of the list, make URL relative, and make the note render correctly...

```suggestion
1. Launch the Arctype desktop client.

1. Follow the in-app prompts to create and log into your Arctype account.

1. On the "Connect a Database" step, select YugabyteDB.

    ![Connect DB](/images/develop/tools/arctype/arctype-connect_step3.png)

    {{< note title="Note" >}}

If you're using YugabyteDB Cloud, you need to add your computer to the cluster IP allow list. Refer to [Assign IP Allow Lists](../../yugabyte-cloud/cloud-secure-clusters/add-connections/). You also need to download and install CA Cert root.crt certificate on your computer from YugabyteDB Cloud console for TLS encryption.

    {{< /note >}}

1. Enter your YugabyteDB host, port, database, user, and password information, and click 'Test Connection' and save if connection is successful.

    ![Enter host and port](/images/develop/tools/arctype/arctype-connect-step4.png)

1. You can see the schemas and tables available in the YugabyteDB in the navigation panel.
```

**Review by polarweasel — APPROVED** [review]
Looks great. One change added, and then it's good to merge.

**Pull request merged**